### ISSUE TITLE
Make the FLashcards page

### ISSUE THREAD

**Issue by MarijaIgnj** [AccessibleAI-edu/accessibleai-edu#36]

**Comment by MarijaIgnj** [comment]
@mutharasus 

Can this white frame around the flashcards be a little thinner? Also, when you click on the flashcards, the box with the explanation is not aligned properly. Close button should be Read more button, as we will have longer articles for all the Flashcards at some point.

One more thing, can you change the visual on the second flashcard (Artificial General Intelligence), I must have sent you a wrong one.

![Image](https://github.com/AccessibleAI-edu/accessibleai-edu/assets/140619642/11c537ce-f7e3-4775-9284-f5d65a67f943)

**Comment by mutharasus** [comment]
@MarijaIgnj

I can make the white border smaller and change the second image to the above image.
Can you please send a screenshot of the alignment issue.
When switching button to be "Read More" shall I make the button show on the screen but disabled for now? Or remove the close button now and add Read More button later when ready ?

**Comment by MarijaIgnj** [comment]
@mutharasus 

Right, you can remove it for now, we will add the Read more button once we have the articles.

Here is the screenshot:



![Image](https://github.com/AccessibleAI-edu/accessibleai-edu/assets/140619642/c0c4fcd4-bdd9-49fd-9433-f6fa0f8a9d2f)

**Comment by MarijaIgnj** [comment]
@mutharasus 

Also, the box with the explanation should be connected with its flashcard. It should not follow the cursor as you scroll on the page. Especially when we remove the Close button.

**Comment by mutharasus** [comment]
@MarijaIgnj  This is how it look in my devices. Can you share your device type. Might be device specific issue. 

![Image](https://github.com/AccessibleAI-edu/accessibleai-edu/assets/134731327/a65f9779-ffcf-4eaa-b285-02ceba051c04)

**Comment by MarijaIgnj** [comment]
@mutharasus I have tried it on a 15 inch laptop. Shouldn't it be optimized for all types of devices?

**Comment by mutharasus** [comment]
@MarijaIgnj  I am testing on 15'', 13'', 21'' monitor sizes. Chrome, Brave and Safari browser on MacOS, Safari & Chrome browsers on iOS. I can recreate the misalignment if I resize the browser after clicking or if I scroll the screen after clicking though. 

These are very difficult to solve because I have to compute the flashcard location constantly and will cause browser performance issues.

The current way it shows the moment you click anywhere on the screen the flashcard is turned over and clicking on the next flash card shows it in the right spot. I figured it was critical that the flashcard opens in the exact spot expected when clicked but it would not be bad UX to stay put when scrolled or resize the screen. 

If it shows in the wrong spot when you click let me know the device type (Operating System and Browser) so I can check if there is some issue in computing the location in javascript that is device specific. The screen size itself I don't think is the problem.

**Comment by mutharasus** [comment]
Most of the changes are made. Please let me know your device type so I can review the Javascript for alignment issue.

**Comment by MarijaIgnj** [comment]
@mutharasus  Well, Ι'm using Windows 10, I don't know what else info do you need?

**Comment by mutharasus** [comment]
@MarijaIgnj let me know the browser type as well. Thanks.

**Comment by MarijaIgnj** [comment]
@mutharasus I'm using Chrome.

**Comment by mutharasus** [comment]
I think we may need to get on a zoom call so I can debug the problem remotely and evaluate what needs to change for Windows 10 - Chrome.

**Comment by MarijaIgnj** [comment]
@mutharasus Yeah, we can do it this week.
